packages/salesforcedx-utils-vscode/src/auth/authUtil.ts

```diff
@@ -24,9 +24,7 @@ export class AuthUtil {
     enableWarning: boolean
   ): Promise<string | undefined> {
     try {
-      const defaultUserName = await ConfigUtil.getConfigValue(
-        DEFAULT_USERNAME_KEY
-      );
+      const defaultUserName = await ConfigUtil.getDefaultUsernameOrAlias();
```

Comment:
Should be able to remove the default_username_key import up above on line 11 now

Reply:
good eye